spec/features/data_access/background_ordering_spec.rb

```diff
@@ -4,8 +4,41 @@
 describe 'Background jobs ordering', reset: false do
   orderable_collection_id = 'C90762182-LAADS'
   orderable_collection_title = 'MODIS/Aqua Calibrated Radiances 5-Min L1B Swath 250m V005'
+  aster_collection_id = 'C14758250-LPDAAC_ECS'
 
-  before :all do
+  context "ordering ASTER data" do
+    before :all do
+      Delayed::Worker.delay_jobs = true
+
+      load_page :search, {project: [aster_collection_id], view: :project}
```

Comment:
I think the fixtures will end up actually trying to place the order. You should prevent them from doing this by using hand edited fixtures.